zips/zip-0226.rst

```diff
@@ -469,21 +560,21 @@ Other Considerations
 Transaction Fees
 ----------------
 
-The fee mechanism for the upgrades proposed in this ZIP will follow the mechanism described in ZIP 317 for the ZSA protocol upgrade [#zip-0317b]_.
+The fee mechanism for the upgrades proposed in this ZIP will follow the mechanism described in ZIP 317 for the OrchardZSA protocol upgrade, and are described in ZIP 230 [#zip-0230-orchardzsa-fee-calculation]_.
```

Comment:
```suggestion
The fee mechanism for the upgrades proposed in this ZIP will follow the mechanism described in ZIP 317 for the OrchardZSA protocol upgrade, and are described in ZIP 230 [#zip-0227-orchardzsa-fee-calculation]_.
```

Reply:
Oops, missed that. Fixed in https://github.com/QED-it/zips/pull/96, thanks!